src/metadata.cpp

```diff
@@ -1367,7 +1417,7 @@ void Metadata::InternalData::update_user_types(ResultResponse* result) {
       fields.push_back(UserType::Field(field_name, data_type));
     }
 
-    get_or_create_keyspace(keyspace_name)->add_user_type(
+    keyspace->add_user_type(
```

Comment:
Same (for the most part); recommend `KeyspaceMetadata* keyspace = get_or_create_keyspace(keyspace_name);` or initializing keypsace to NULL.